spec/oca-epak/oep/client_spec.rb

```diff
@@ -33,4 +33,19 @@
       end
     end
   end
+
+  describe "#get_pdf_de_etiquetas_por_orden_or_numero_envio" do
+    let(:real_order) { 21132466 }
+    let(:fake_order) { 123_000_000 }
```

Comment:
Again sorry to left this here without the proper spec. Going to add that scenario